README.md

```diff
@@ -176,28 +224,30 @@ Spectrum CSS is designed to be as flexible as possible, and as such, leaves room
 You've created an `index.css` that imports a few components, a scale, and a color-theme:
 
 ```css
-@import 'node_modules/@spectrum-css/vars/dist/spectrum-global.css';
-@import 'node_modules/@spectrum-css/vars/dist/spectrum-medium.css';
-@import 'node_modules/@spectrum-css/vars/dist/spectrum-light.css';
-@import 'node_modules/@spectrum-css/page/dist/index-vars.css';
-@import 'node_modules/@spectrum-css/button/dist/index-vars.css';
+@import "node_modules/@spectrum-css/vars/dist/spectrum-global.css";
```

Comment:
Is there any specific reason we have updated the single quote to double quotes?
Can we go by a unified lint rule approach?